apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/SpansResourceTest.java

```diff
@@ -1058,6 +1058,65 @@ void getByProjectName__whenFilterIdAndNameEqual__thenReturnSpansFiltered() {
             getAndAssertPage(workspaceName, projectName, filters, spans, expectedSpans, unexpectedSpans, apiKey);
         }
 
+        @Test
```

Comment:
Let's use ParameterizedTest instead of testing many scenarios in one tests

Reply:
Updated